Rx/v2/src/rxcpp/operators/rx-merge_delay_error.hpp

```diff
@@ -0,0 +1,302 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved. See License.txt in the project root for license information.
+
+#pragma once
+
+/*! \file rx-merge_delay_error.hpp
+
+        \brief For each given observable subscribe.
+                   For each item emitted from all of the given observables, deliver from the new observable that is returned.
+				   The first error to occure is hold off until all of the given non-error-emitting observables have finished their emission.
+
+                   There are 2 variants of the operator:
+                   - The source observable emits nested observables, nested observables are merged.
+                   - The source observable and the arguments v0...vn are used to provide the observables to merge.
+
+        \tparam Coordination  the type of the scheduler (optional).
+        \tparam Value0	... (optional).
+        \tparam ValueN	types of source observables (optional).
+
+        \param	cn		the scheduler to synchronize sources from different contexts (optional).
+        \param	v0		... (optional).
+        \param	vn		source observables (optional).
+
+        \return																											 Observable that emits items that are the result of flattening the observables emitted by the source observable.
+
+        If scheduler is omitted, identity_current_thread is used.
+
+        \sample
+        \snippet merge_delay_error.cpp threaded implicit merge sample
+        \snippet output.txt threaded implicit merge sample
+
+        \sample
+        \snippet merge_delay_error.cpp implicit merge sample
+        \snippet output.txt implicit merge sample
+
+        \sample
+        \snippet merge_delay_error.cpp merge sample
+        \snippet output.txt merge sample
+
+        \sample
+        \snippet merge_delay_error.cpp threaded merge sample
+        \snippet output.txt threaded merge sample
+*/
```

Comment:
These examples need to be defined in a new file next to https://github.com/Reactive-Extensions/RxCpp/blob/master/Rx/v2/examples/doxygen/merge.cpp

You might want to have an example that demonstrates the `composite_error`

Reply:
I added those doxygen examples:
* https://github.com/nitrram/RxCpp/blob/merge_error_delay/Rx/v2/examples/doxygen/merge_delay_error.cpp
* https://github.com/nitrram/RxCpp/blob/merge_error_delay/Rx/v2/examples/doxygen/composite_exception.cpp

-- hopefully sufficiently enough

Yet if there is something missing, let me know.

Reply:
looks great! I did not see these added to https://github.com/Reactive-Extensions/RxCpp/blob/master/projects/doxygen/CMakeLists.txt 

did I miss that?

Reply:
Ah, my bad! I forgot that. I should also change the documentation at rx-merge_delay_error. cpp so that it matches the tests which I'm afraid does not at the moment. Unfortunately I'll be at a computer not sooner than on Sunday. But I'll fix it right then. Thanks for the review and patience

Reply:
Ok, so eventually I managed that. I added those 2 samples among the others. The documentation turned to be fine to me after all. When it's approved, shall I do something about it (squash commits...), before it gets merged?